CommonWallet/Core/Classes/Modules/HistoryFilter/HistoryFilterPresenter.swift

```diff
@@ -15,18 +15,18 @@ private enum DateSelection: Int {
 }
 
 
-final class FilterPresenter {
-    weak var view: FilterViewProtocol?
-    var coordinator: FilterCoordinatorProtocol
+final class HistoryFilterPresenter {
+    weak var view: HistoryFilterViewProtocol?
+    var coordinator: HistoryFilterCoordinatorProtocol
```

Comment:
coordinator is unused?

Reply:
It is used. Just changed name of the type so gh doesn't show places where the variable is used.